ballerina/Module.md

```diff
@@ -1,7 +1,32 @@
-## Overview
-Salesforce REST API provides CRUD operations for SObjects, query using SOQL, search using SOSL, and describe SObjects and organizational data.
+## Package overview
+Salesforce Sales Cloud is one of the leading Customer Relationship Management(CRM) software, provided by Salesforce.Inc. Salesforce enable users to efficiently manage sales and customer relationships through its APIs, robust and secure databases, and analytics services. Sales cloud provides serveral API packages to make operations on sObjects and metadata, execute queries and searches, and listen to change events through API calls using REST, SOAP, and CometD protocols. 
```

Comment:
A newlinw after headings

Reply:
Updated in [6df510d](https://github.com/ballerina-platform/module-ballerinax-sfdc/pull/311/commits/6df510d1098d680d7b11812ca4ebd7b34b2af897)